Automattic/newspack-theme
# fix: es_ES translation for "time ago" string

### All Submissions:






### Changes proposed in this Pull Request:



Updates the Spanish translation for the "time ago" string.

### How to test the changes in this Pull Request:

1. On `master`, set your site language to "Español". Ensure that the "Use 'time ago' date format" option is enabled in Customizer > Template Settings > Post Settings.
2. View a singular post published or updated within the past 24 hours. Observe that the date is shown as Spanglish, e.g. `15 horas ago`
3. Check out this branch, refresh the page, confirm that it's now translated as `15 horas antes`

### Other information:
